website/src/client/components/ThemeProvider.tsx

```diff
@@ -142,3 +135,16 @@ const styles = StyleSheet.create({
     ...mapKeys(darkShadows, (_, key) => `--shadow-${key}`),
   },
 });
+
+function convertToHex(color: string) {
+  if (color.startsWith('hsl')) {
+    const hslChunks = color
+      .replace('hsl(', '')
+      .replace(')', '')
+      .replace('%', '')
+      .split(',')
+      .map((c) => parseInt(c.trim(), 10));
```

Comment:
I think using a RegExp might be better here, just to support slightly different notations (such as `hsl (...)` or rogue spaces).

From what I could find online, [this regex seems to do exactly that](https://regex101.com/r/ZSUEig/1) - based on this lib: https://github.com/regexhq/hsl-regex/blob/master/index.js#LL8C5-L8C67 (with some slight adjustments)